python/raydp/context.py

```diff
@@ -126,12 +123,10 @@ def stop(self, del_obj_holder=True):
         if self._spark_session is not None:
             self._spark_session.stop()
             self._spark_session = None
-        if self.handle is not None and del_obj_holder is True:
-            self.handle.terminate.remote()
-            self.handle = None
         if self._spark_cluster is not None:
-            self._spark_cluster.stop()
-            self._spark_cluster = None
+            self._spark_cluster.stop(del_obj_holder)
```

Comment:
The API `raydp.stop_spark(del_obj_holder=True)` is user facing. Any idea for a better name to replace name `del_obj_holder`?

Reply:
Maybe `raydp.stop_spark(cleanup_data=True)`?

Reply:
I am thinking shall we give it a default value?

Reply:
Hmm, giving a default value keeps the behavior same as before. Otherwise users might upgrade raydp and find out the scripts are failing.

Reply:
right, but I think we don't have a default right now, or am i missing something?

Reply:
Yes, we have a default value now, we're talking if we should change the option name

Reply:
It sounds better. Please also add some document to avoid confusion.